sympy/combinatorics/perm_groups.py

```diff
@@ -3461,24 +3528,38 @@ def _orbit_transversal(degree, generators, alpha, pairs, af=False):
     """
 
     tr = [(alpha, list(range(degree)))]
+    slp_dict = {alpha: []}
     used = [False]*degree
     used[alpha] = True
     gens = [x._array_form for x in generators]
     for x, px in tr:
+        px_slp = slp_dict[x]
         for gen in gens:
             temp = gen[x]
             if used[temp] == False:
+                slp_dict[temp] = [gens.index(gen)] + px_slp
+                w = list(range(degree))
+                for s in slp_dict[temp]:
+                    w = _af_rmul(w,gens[s])
```

Comment:
No, I think this is left over from when I was making sure things worked properly. I'll remove it.